installer.sh

```diff
@@ -43,6 +43,15 @@ MIN_PYZMQ_VERSION="14.4"
 MIN_PYCRYPTOGRAPHY_VERSION="2.1.4"
 MIN_GO_VERSION="1.8.4"
```

Comment:
We might want to consider bumping go's minimum to `1.11.13`, since `1.8.4` is pretty old there are various open CVEs for anything before that.

Reply:
Also, from above, the current version is `1.13.1`.  Anything before `1.12.8` has an open CVE.  It would be good to at least bump this to `1.12.10`.